test/performance/tools/common.sh

```diff
@@ -48,66 +45,58 @@ function create_cluster() {
     --scopes cloud-platform
 }
 
-# Copies serice account secret to the destination
-# $1 -> destination context. Should be of the form ${CONTEXT_PREFIX}_${zone}_${name}
-function copy_secret() {
-  gcloud container clusters get-credentials ${MASTER_CLUSTER_NAME} --project=${PROJECT_NAME} --zone=${MASTER_CLUSTER_ZONE}
-  kubectl get secret service-account --context ${SOURCE_CONTEXT} --export -o yaml \
-    | kubectl apply --context $1 -f -
+# Create serice account secret on the cluster.
+# $1 -> cluster_name, $2 -> cluster_zone
+function create_secret() {
+  name=$1
```

Comment:
no need to create these vars. The method documentation mentions the params

Reply:
Fixed